WcaOnRails/app/controllers/api/v0/api_controller.rb

```diff
@@ -9,9 +10,17 @@ def bad_api_parameter(e)
 
   DEFAULT_API_RESULT_LIMIT = 20
 
+  def current_resource_owner
+    if doorkeeper_token
+      User.find_by_id(doorkeeper_token.resource_owner_id).tap do |user|
+        user.doorkeeper_token = doorkeeper_token
```

Comment:
Gah, sorry! There should be `&.` before `tap` of course. I don't feel unwell with `User.find_by_id(nil)` saving some lines =)